src/store/reducers/describe.ts

```diff
@@ -59,26 +86,28 @@ const describe: Reducer<
 };
 
 // Consumers selectors
-const selectConsumersNames: Selector<IDescribeRootStateSlice, string[] | undefined, [string]> = (
-    state,
-    path,
-) => state.describe.data[path]?.PathDescription?.PersQueueGroup?.PQTabletConfig?.ReadRules;
-
-export const selectConsumers: Selector<IDescribeRootStateSlice, IConsumer[]> = createSelector(
-    selectConsumersNames,
-    (names = []) => {
+const selectConsumersNames = (state: IDescribeRootStateSlice, path: string | undefined) =>
+    path
```

Comment:
You may skip this check because `someArray[undefined] = undefined`.

Reply:
It was done to satisfy TypeScript